wp-content/themes/core/components/blocks/card_grid/Card_Grid_Controller.php

```diff
@@ -76,7 +76,7 @@ protected function defaults(): array {
 	protected function required(): array {
 		return [
 			self::CONTAINER_CLASSES => [ 'b-card-grid__container', 'l-container' ],
-			self::LOOP_CLASSES      => [ 'b-card-grid__loop' ],
+			self::LOOP_CLASSES      => [ 'b-card-grid__loop', 't-theme--light' ],
```

Comment:
@ScottGarrison, Curious why the t-theme--light class is necessary, especially as a required class. Is this the default anyway?

Reply:
because of the hierarchy. If the block itself gets `t-theme--dark`, then the card itself gets a black background with white text. It looked a little strange that way so I just override it here so the block itself gets the black background but the cards are the normal white background with black text. It might not need to be part of the "required" classes